uktrade/data-hub-frontend
# Make UK region filter a type ahead on companies collection

## Description of change

This pull request updates the UK Regions filter in the company's collection page to be a type ahead, rather than check boxes. 

The original uk region functional test (in companies) has been changed to test the new type ahead ability. 

## Test instructions

- Go to the companies collection page
- Filter UK Region by North West

_What should I see?_

## Screenshots
### Before

![Screenshot 2020-01-16 at 13 49 55](https://user-images.githubusercontent.com/22460823/72530350-1696f400-3867-11ea-9ea1-67d7259bc864.png)

### After

![Screenshot 2020-01-16 at 13 50 34](https://user-images.githubusercontent.com/22460823/72530381-2d3d4b00-3867-11ea-9460-6940f7ae6082.png)


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
